posthog/api/capture.py

```diff
@@ -138,8 +139,8 @@ def get_event(request):
             ip=get_ip_address(request),
             site_url=request.build_absolute_uri('/')[:-1],
             data=event,
-            team_id=team_id,
-            now=now,
+            team=team,
```

Comment:
I'm surprised the serialisation and sending it over redis entirely is quicker than what we were currently doing. It might also clog Redis itself

Reply:
Clogging redis is a real potential problem. We will need to watch that.

The serialization doesn't surprise me too much. Django has to build and serialize the query, wait for db, and then unmarshal the results from postgres anyways. This just pulls the query building and network out of the loop.

Reply:
Clogging redis has already been a problem with a few of the big Heroku deployments I looked at, so why don't we do this caching in process_event instead? The Team object will basically always be the same anyway for those instances (as there's only one team). That way we don't have to send  this over Redis